tests/payments.py

```diff
@@ -40,4 +42,18 @@ def test_create_payment_type(self):
         self.assertEqual(json_response["expiration_date"], "2024-12-31")
         self.assertEqual(json_response["create_date"], str(datetime.date.today()))
 
-    # TODO: Delete payment type
\ No newline at end of file
+
+    def test_delete_payment_type(self):
+        """
+        Ensure we can delete a payment type.
+        """
+        # Add product to order
+        self.test_create_payment_type()
```

Comment:
Each test function should be able to do its job without relying on other test functions. This `test_create_payment_type` function's job is to test the `create()` method in your PaymentView class; not to seed your database, and it should not be used to create seed data in your database.

Instead, use your Payment class to instantiate a new payment object, and invoke the .save() method to add it to your database. That way, your `test_delete_payment_type` function will still work, even if your `test_create_payment_type` function breaks.